LoRaEngine/LoraKeysManagerFacade/CreateEdgeDevice.cs

```diff
@@ -82,22 +85,46 @@ public CreateEdgeDevice(RegistryManager registryManager)
             try
             {
                 _ = await this.registryManager.AddDeviceAsync(edgeGatewayDevice);
+                _ = await this.registryManager.AddModuleAsync(new Module(deviceName, "LoRaWanNetworkSrvModule"));
 
-                var deviceConfigurationUrl = Environment.GetEnvironmentVariable("DEVICE_CONFIG_LOCATION");
-                string json = null;
-
-                // todo correct
-                using (var wc = new WebClient())
+                static ConfigurationContent GetConfigurationContent(string configLocation, IDictionary<string, string> tokenReplacements)
                 {
-                    json = wc.DownloadString(deviceConfigurationUrl);
+                    // todo correct
```

Comment:
What is meant by this todo?

Reply:
This was already there, I just made sure to not delete an existing comment (even though I don't know the context).

Reply:
Still went ahead and removed the comment in d18d690e1aa431181cf29355b850be6ab36fe434